buildpack.toml

```diff
@@ -42,22 +42,32 @@ api = "0.7"
     id = "jattach"
     name = "JAttach"
     purl = "pkg:generic/jattach@2.2.0?arch=amd64"
-    sha256 = "a08cb795a1e8d11ea6c2dd6adf8c9edead9a7c3bbca07681dad79cc3eaec0ef4"
+    sha256 = "acd9e17f15749306be843df392063893e97bfecc5260eef73ee98f06e5cfe02f"
     source = "https://github.com/apangin/jattach/archive/refs/tags/v2.2.tar.gz"
     source-sha256 = "dc29ebb4fe795a026698fd77d04072cfb39295276a541cfc5e77d20b1509a1cf"
-    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
-    uri = "https://github.com/jattach/jattach/releases/download/v2.2/jattach"
+    stacks = ["*"]
+    uri = "https://github.com/jattach/jattach/releases/download/v2.2/jattach-linux-x64.tgz"
```

Comment:
so what's the difference between `jattach` and `jattach-linux-x64.tgz` ?

Reply:
If you extract the tgz, you get `jattach`. The sha hash is the same.